api/v1/submissions/PKPSubmissionHandler.php

```diff
@@ -478,35 +478,35 @@ public function add(SlimRequest $slimRequest, APIResponse $response, array $args
         if (isset($params[$sectionIdPropName])) {
             $sectionId = $params[$sectionIdPropName];
             $section = Repo::section()->get($sectionId, $context->getId());
-            if ($section->getIsInactive()) {
-                $errors[$sectionIdPropName] = [__('api.submission.400.inactiveSection')];
+            if (!$section) {
+                $errors[$sectionIdPropName] = [__('api.submission.400.sectionDoesNotExist')];
```

Comment:
Maybe adding a new locale here is not such a good idea for stable branch - I tried to find a more generic error but did not find something suitable.